workflows/reference_files/1.0/config/default.yaml

```diff
@@ -1,3 +1,5 @@
+reference_directory: "reference/"
```

Comment:
Where is this required? Were you running into an error if this was omitted?

Reply:
No that was from when I was trying to create a rule and didn't realize the working directory is already in `reference/`